ert_storage/database_schema/ensemble.py

```diff
@@ -44,5 +42,9 @@ class Ensemble(Base, MetadataField):
     )
 
     @property
-    def parameters(self) -> Iterable["Record"]:
-        return self.records.filter_by(record_class=RecordClass.parameter)
+    def parent_ensemble_id(self) -> PyUUID:
```

Comment:
Why we need these `_id` type properties?

Reply:
It's used in the pydantic models, where we can specify
```py3
parent: Optional[UUID] = Field(alias="parent_ensemble_id")
```
This way we don't have to manually construct the pydantic models in the endpoints, where instead we can simply return the SQLAlchemy objects, avoding code duplication.